cinema_service/urls.py

```diff
@@ -1,9 +1,12 @@
+from django.conf.urls.static import static
 from django.contrib import admin
+from django.conf import settings
 from django.urls import path, include
 
+
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `urls.py` files in the `cinema` and `user` apps define an `app_name` variable. This is required when using the `namespace` argument in the `include` function.